include/engine/phantom_node.hpp

```diff
@@ -78,7 +76,7 @@ struct PhantomNode
           reverse_segment_id{SPECIAL_SEGMENTID, false},
           name_id(std::numeric_limits<unsigned>::max()), forward_weight(INVALID_EDGE_WEIGHT),
           reverse_weight(INVALID_EDGE_WEIGHT), forward_offset(0), reverse_offset(0),
-          forward_packed_geometry_id(SPECIAL_EDGEID), reverse_packed_geometry_id(SPECIAL_EDGEID),
+          packed_geometry_id(SPECIAL_EDGEID),
```

Comment:
Whatever we use, we should be consistent with SPECIAL_EDGEID to be assigned to edge_ids, geometry_ids only using a SPECIAL_GEOMETRYID.